packages/theme-a11y/theme-a11y.js

```diff
@@ -264,7 +264,8 @@ export function accessibleLinks(elements, options) {
     var isRelNoopenerEmpty = rel === null || rel.indexOf('noopener') === -1;
```

Comment:
Can we rename this var to something like `missingRelNoopener`? Current name confused me for a second.

Reply:
For sure :)